tests/structures/test_generator.py

```diff
@@ -72,24 +72,48 @@ def fizz_buzz(self):
                 print(i)
             """)
     
-    def test_yield_from_not_used(self):
-        """Yield from is currently not implemented.
-        Unused yield from statements must not break 
-        the program compilation, but only ensue a warning."""
+    def test_simplest_yieldfrom(self):
         self.assertCodeExecution("""
-            
-            def unused():
-                yield from range(5)
+            def gen1():
+                yield 1
+                yield 2
+                yield 3
+                
+            def gen2():
+                yield from gen1()
+                
+            for i in gen2():
+                print(i)
+        """)
 
-            print('Hello, world!')
-            """)
+    def test_yieldfrom_list(self):
+        self.assertCodeExecution("""
+            def gen():
+                yield from [1, 2, 3]
+                
+            for i in gen():
+                print(i)
+        """)
+
+    def test_chainning_yieldfrom(self):
+        self.assertCodeExecution("""
+            def gen1():
+                yield 'a'
+                yield 'b'
+                
+            def gen2():
+                yield from gen1()
+                yield 'gen2'
+                
+            def gen3():
+                yield from gen2()
+                
+            for i in gen3():
+                print(i)
+        """)
     
-    @expectedFailure
     def test_yield_from_used(self):
```

Comment:
This is one of those rare situations where a test can be removed. The test itself is a much simpler version of what you've tested thoroughly with your other `yield from` tests.